app/config.py

```diff
@@ -35,7 +54,7 @@ def get_abs_path(file_path: str):
 PROMO_CODE = "SIMPLEISBETTER"
 
 # Debug mode
-DEBUG = os.environ["DEBUG"] if "DEBUG" in os.environ else False
+DEBUG = os.environ.get("DEBUG", default=False)
```

Comment:
This can return a non-boolean value of DEBUG (which we don't want)

Reply:
Indeed, but I think the old way has the same problem. I've fixed on the latest push.